components/backends/vllm/src/dynamo/vllm/handlers.py

```
@@ -146,6 +146,10 @@ async def generate(self, request, context):
         for key, value in request["stop_conditions"].items():
             if value is not None and hasattr(sampling_params, key):
                 setattr(sampling_params, key, value)
+            if key == "stop_token_ids_hidden" and hasattr(
+                sampling_params, "stop_token_ids"
+            ):
+                setattr(sampling_params, "stop_token_ids", value)
```

Ruff (0.13.1)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)